_pkgdown.yml

```diff
@@ -68,6 +68,11 @@ reference:
   contents:
   - has_keyword('timing')
 
+- subtitle: High Order Functions
+  desc: Functions which call other derivations, e.g., on subset of input dataset or multiple times with varying parameters.
```

Comment:
There should be no `,` after "e.g.".

Reply:
I think it depends on the style guide. It seems that American style guides prefer the comma while British one doesn't.

I have removed it.